templates/build/agenda-cards-loop.build.hbs

```diff
@@ -19,7 +19,7 @@
                   \{{#if [Location]}}
                   <p class="agenda-item-location">\{{{[Location]}}}</p>
                   \{{/if}}
-                  <p class="agenda-item-start-date-time">\{{[Start Time]}}\{{#ifCond [Start Time] '&&' [End Time]}} - \{{/ifCond}}\{{[End Time]}}\{{#if [Date]}} / \{{/if}}\{{[Date]}}</p>
+                  <p class="agenda-item-start-date-time">\{{[Start Time]}}\{{#ifCond [Start Time] '&&' [End Time]}} - \{{/ifCond}}\{{[End Time]}}\{{#if [Full Date]}} / \{{/if}}\{{formatDate [Full Date]}}</p>
```

Comment:
Hopefully you won't need this soon because we're working on the `moment` helper. That would help, right?

Reply:
@tonytlwu I won't need that helper, I'll update to the one you create(d).